test/mockRaf.js

```diff
@@ -0,0 +1,35 @@
+export default {
+  allCallbacks: [],
+  prevTime: 0,
+
+  getNow() {
+    return () => {
+      if (this.prevTime === 0) {
+        this.prevTime = Date.now();
```

Comment:
Ha! turns out we don't need to set the value to something different than 0, we're only using it for delta time right now :D